packages/gatsby/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@sentry/gatsby",
+  "version": "5.16.1",
+  "description": "Offical Sentry SDK for Gatsby.js",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/gatsby",
+  "author": "Sentry",
+  "license": "BSD-3-Clause",
+  "engines": {
+    "node": ">=6"
+  },
+  "main": "dist/index.js",
+  "module": "esm/index.js",
+  "types": "dist/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@sentry/browser": "5.16.1",
+    "@sentry/types": "5.16.1"
+  },
+  "peerDependencies": {
+    "gatsby": "*"
```

Comment:
This config will work for all gatsby versions?

Reply:
I dont know what versions itll work for, and I don't really want to dig through Gatsby history to figure out when specific APIs were created/etc.